student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, name, parent_permission, classroom)
```

Comment:
- [ ] It will be great to ensure `name` and `parent_permission` have their default values set as `name = "Unknown", parent_permission = true` respectively

Reply:
Hello @peterrobert , i thought that the name and the parent_permission inherits from class Person, so that is why I did not put the default values in both, is that correct.